src/pairs.rs

```diff
@@ -257,6 +269,7 @@ trait FlattenPair: Rewrite + Sized {
 struct PairList<'a, 'b, T: Rewrite> {
     list: Vec<(&'b T, Option<String>)>,
     separators: Vec<&'a str>,
+    span: Span,
 }
```

Comment:
I simply set the span of `PairList` as `Expr.span` in `flatten()`, but I wonder if I should grab the span with looking at the `T.span` of first item and the last item of `list` field

Reply:
Should be fine to use the `expr.span` since that encompasses the entire BinaryExpr, which is represented as a tree in the AST. I believe flatten is just trying to transform that tree structure into a flat list.